package.json

```diff
@@ -83,17 +83,17 @@
     "node": ">=16.20"
   },
   "dependencies": {
-    "@cdktf/provider-archive": "4.0.0",
-    "@cdktf/provider-aws": "11.0.9",
-    "@cdktf/provider-local": "4.0.0",
-    "@cdktf/provider-newrelic": "5.0.3",
-    "@cdktf/provider-null": "4.0.1",
-    "@cdktf/provider-pagerduty": "4.0.3",
-    "@cdktf/provider-time": "4.0.0",
+    "@cdktf/provider-archive": "^7.0.0",
+    "@cdktf/provider-aws": "^16.0.2",
+    "@cdktf/provider-local": "^7.0.0",
+    "@cdktf/provider-newrelic": "^9.0.0",
+    "@cdktf/provider-null": "^7.0.0",
+    "@cdktf/provider-pagerduty": "^8.0.0",
+    "@cdktf/provider-time": "^7.0.0",
     "@sinonjs/commons": "^3.0.0",
-    "cdktf": "0.14.3",
-    "cdktf-cli": "0.14.3",
-    "constructs": "10.1.215",
+    "cdktf": "^0.17.1",
+    "cdktf-cli": "^0.17.1",
+    "constructs": "^10.2.69",
```

Comment:
for safety's sake, do we want to keep pinning to specific versions? what's our philosophy here?

Reply:
@jpetto, I can manually take the carets out or wait for Renovate to create PRs that pin the dependencies to the exact minor version - we've done the latter many times